src/DIRAC/RequestManagementSystem/private/RequestTask.py

```diff
@@ -403,7 +403,8 @@ def __call__(self):
                             if operation.Status != "Failed":
                                 operation.Status = "Failed"
                             self.request.Error = "Job no longer exists"
-            except Exception as error:
+            except Exception as e:
+                error = str(e)
                 self.log.exception("hit by exception:", "%s" % error)
```

Comment:
I'm pretty sure this is the same:
```suggestion
            except Exception as error:
                self.log.exception("hit by exception:", error)
```

Reply:
yes, except that `error` is reused later. See  the original PR description. It is really a weird edge case in my opinion :-)

Reply:
Ooops, this was merged!